airbyte-db/lib/src/main/resources/configs_database/schema2.sql

```diff
@@ -0,0 +1,218 @@
+CREATE
+    TYPE "source_type" AS enum(
+        'api',
+        'file',
+        'database',
+        'custom'
+    );
+
+CREATE
+    TYPE "actor_type" AS enum(
+        'source',
+        'destination'
+    );
+
+CREATE
+    TYPE "operator_type" AS enum(
+        'normalization',
+        'dbt'
+    );
+
+CREATE
+    TYPE "namespace_definition_type" AS enum(
+        'source',
+        'destination',
+        'customformat'
+    );
+
+CREATE
+    TYPE "status_type" AS enum(
+        'active',
+        'inactive',
+        'deprecated'
+    );
+
+CREATE
+    TABLE
+        IF NOT EXISTS "workspace"(
```

Comment:
we should be careful about creating consistency in columns names. that will be use as an example in the future.